js/components/graph/Container.js

```diff
@@ -6,11 +6,15 @@ import PropTypes from "prop-types";
 // import * as sidebarDivs from "./sidebar/sidebar_divs.js";
 
 export default class Container extends React.Component {
+  constructor(props) {
+    super(props);
+    this.graphRef = React.createRef;
+  }
   render() {
     return (
       <React.Fragment>
-        <Graph start_blank={this.props.start_blank} edit={this.props.edit} initialDrawMode="draw-node" />
-        <Sidebar />
+        <Graph start_blank={this.props.start_blank} edit={this.props.edit} initialDrawMode="draw-node" ref={instance => {this.graphRef = instance; }}/>
```

Comment:
Because we are using a ref for this component rather locally at the render level, you can do `ref={this.graphRef}` or else our ref isn't being used.